### PR TITLE
[IMP] account_move_report: black, isort, prettier

### PR DESCRIPTION
**Pull request by ajqn9094**
Apply pre-commit-vauxoo to reformat the code

### CODE REVIEW

**Comment by ajqn9094** [comment]
@luisg123v Could you review please.

**Comment by ajqn9094** [comment]
@luisg123v Could you review please.

File: account_move_report/__manifest__.py [hunk 1]
```diff
@@ -9,10 +9,7 @@
         "account",
     ],
     "demo": [],
-    "data": [
-        "views/account_reports.xml",
-        "views/account_move_report.xml"
-    ],
+    "data": ["views/account_reports.xml", "views/account_move_report.xml"],
```
Comment (luisg123v) [inline account_move_report/__manifest__.py]: Add a trailing comma so one item per line is preserved

File: account_move_report/views/account_move_report.xml [hunk 2]
```diff
@@ -32,39 +41,56 @@
...
+                                            groups="analytic.group_analytic_accounting"
+                                        ><small t-field="l.analytic_account_id" /></td>
                                         <t t-if="l.currency_id">
                                             <td name="td_amount_currency" class="text-right">
-                                                <small t-field="l.amount_currency"/>
+                                                <small t-field="l.amount_currency" />
                                             </td>
                                         </t>
-                                        <td name="td_currency"><small t-field="(l.currency_id or l.company_id.currency_id).name"/></td>
+                                        <td name="td_currency"><small
```
Comment (luisg123v) [inline account_move_report/views/account_move_report.xml]: Indentation

File: account_move_report/views/account_move_report.xml [hunk 3]
```diff
@@ -32,39 +41,56 @@
...
                                     </tr>
                                 </tbody>
                                 <tfoot>
                                     <tr>
-                                        <td colspan="5"></td>
-                                        <td name="td_total_debit" class="text-right"><strong><span t-esc="total_debit_credit[o.id]['sum_tot_debit']" t-options='{"widget": "monetary", "display_currency": o.currency_id}'/></strong></td>
-                                        <td name="td_total_credit" class="text-right"><strong><span t-esc="total_debit_credit[o.id]['sum_tot_credit']" t-options='{"widget": "monetary", "display_currency": o.currency_id}'/></strong></td>
-                                        <td colspan="3"></td>
+                                        <td colspan="5" />
+                                        <td name="td_total_debit" class="text-right"><strong><span
```
Comment (luisg123v) [inline account_move_report/views/account_move_report.xml]: Indentation.

Same in similar cases

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by luisg123v** [comment]
Title:
- black, isort, prettier
- Remove task ID

File: account_move_report/__manifest__.py (hunk 1 above)
Comment (ajqn9094) [reply account_move_report/__manifest__.py]: Done

File: account_move_report/views/account_move_report.xml (hunk 2 above)
Comment (ajqn9094) [reply account_move_report/views/account_move_report.xml]: Done

File: account_move_report/views/account_move_report.xml (hunk 3 above)
Comment (ajqn9094) [reply account_move_report/views/account_move_report.xml]: Done

**Comment by ajqn9094** [comment]
@luisg123v Changes applied

**Review by luisg123v — APPROVED** [review]

**Pull request merged**